superset/commands/dashboard/export.py

```diff
@@ -173,7 +183,14 @@ def _export(
 
         if export_related:
             chart_ids = [chart.id for chart in model.slices]
+            dashboard_ids = model.id
+            ExportChartsCommand.disable_tag_export()
             yield from ExportChartsCommand(chart_ids).run()
+            ExportChartsCommand.enable_tag_export()
```

Comment:
Setting the class attribute can be problematic if we have multiple exports happening at the same time in the same process, there could be a race condtion. It would be better to instantiate the command and set it there to prevent this:

```python
command = ExportChartsCommand(chart_ids)
command.disable_tag_export()
yield from command.run()
command.enable_tag_export()
```

But I think it's better and clearer if we just pass a parameter when we instantiate the command:

```suggestion
            yield from ExportChartsCommand(chart_ids, include_tags=True).run()
```

And then we update `ExportChartsCommand` accordingly:

```python
class ExportChartsCommand(ExportModelsCommand):
    def __init__(self, *args: Any, include_tags: bool = True, **kwargs: Any):
        self._include_tags = include_tags
        super(*args, **kwargs)
```